tests/logging_test.py

```diff
@@ -0,0 +1,34 @@
+"""Test cases for logging related features."""
+
+# Third-Party Libraries
+import pytest
+
+
+class TestLogging:
+    """Test case for logging related views."""
+
+    def test_logging_view(self, client):
+        """Test the logging view."""
+        resp = client.get("/api/logging/")
+        assert resp.status_code == 200
+
+        self.check_logging_properties(resp.json)
+
+    @staticmethod
+    def check_logging_properties(logging):
+        """Check logging object for expected properties."""
+        try:
+            assert isinstance(logging, list)
+        except KeyError:
+            pytest.fail("expected a list")
```

Comment:
this won't work as you expect. I would do this

```
if not isinstance(logging, list):
    pytest.fail("expected a list")
```

there are a few spots in this pr where this is required

Reply:
Much better, thanks! Resolved with e3d9595830337df26d5c91a6890d6364d5eed8fe

Reply:
oops forgot to @xlyk